### ISSUE TITLE
Nested source files are sourcemapped as js instead of ts

### ISSUE THREAD

**Issue by foobardevpro** [theseanl/tscc#82]
1. Download [todomvc-master/typescript-react](https://github.com/theseanl/todomvc/tree/master/typescript-react)
2. In ``tsconfig.json``, under ``compilerOptions`` add line:
```
"sourceMap": true
```
3. Use any source file nested in a folder, for example create ``nested`` folder and create file:
```
// typescript-react/js/nested/test.ts

console.log('test')
```
and add import in ``app.tsx``:
```
import "./nested/test";
```
4. Run tscc (global cli) in /js.
5. Run with your server of choice (I use ``http-server`` cli from npm).
In Google Chrome the console log originates from ``test.js``, instead of ``test.ts`` -- so .ts files are not properly sourcemapped when nested in a folder.

**Comment by theseanl** [comment]
It works on my end, it again looks like to have caused by path handling of Windows.

**Comment by theseanl** [comment]
This could be due to tscc not adding `//# sourceMappingURL=...` comments. If your source files are flat directory-wise, does this issue still happen?

**Comment by foobardevpro** [comment]
No, the issue only happens for nested files. Everything else that is not nested shows up as ts as expected.

**Comment by theseanl** [comment]
Reproduced on my end with Windows machine; maybe this is related to path handling of closure compiler on Windows.

**Comment by theseanl** [comment]
It seems that the compiler only recognizes unix-style paths in inputs -- when windows-style path seperator is used, it fails to resolve sourcemaps. Should be fixed with b24b1d894bb11b803ae7beafde70d8595d6e649e. Please check with 0.4.6!
